query-engine/core/src/executor/request_context.rs

```diff
@@ -18,14 +18,13 @@ tokio::task_local! {
 ///
 /// If we had a query context we carry for the entire lifetime of the query, it would belong there.
 pub(crate) fn get_request_now() -> PrismaValue {
-    // FIXME: we want to bypass task locals if this code is executed outside of a tokio context. As
-    // of this writing, it happens only in the query validation test suite.
-    //
-    // Eventually, this will go away when we have a plain query context reference we pass around.
-    if tokio::runtime::Handle::try_current().is_err() {
```

Comment:
What would be other cases in which the tokio handle would not exist that will make the original code necessary?

It's sketchy to me, not that removing it fixes WASM, but that it doesn't break anything else.

Reply:
Comment there mentions query validation test suite. My guess is that check for runtime was there to avoid a panic on unset `task_local`, not for anything runtime-specific. Which `try_with` handles as well.